src/unibit.js

```diff
@@ -370,6 +370,7 @@ module.exports = class Unibit {
 
     getPageByFilePath(context, filePath) {
         // remove extension
+        filePath = path.relative(UNIBIT.pagesDir, filePath);
         return _.find(context.site.pages, page => {
             const pageUrl = _.get(page, 'relPath');
             return filePath === pageUrl;
```

Comment:
@rodikh don't forget removing obsolete stuff
1. comment `// remove extension`
2. rename `pageUrl` to `pageRelPath`